### PR TITLE
doc(template): add payg section with Algolia template

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

File: templates/payg/algolia.mdx [hunk 1]
```diff
@@ -2,4 +2,279 @@
...
+            "name": "Algolia Starter",
+            "code": "algolia_starter",
+            "interval": "monthly",
+            "amount_cents": 0,
+            "amount_currency": "USD",
+            "trial_period": 0.0,
+            "pay_in_advance": false,
+            "charges": [
+                {
+                    "lago_billable_metric_id": "15879740-102e-43bd-a6bc-15d865bb1b85",
```
Comment (sarkissianraffi) [inline templates/payg/algolia.mdx:110]: This field should be billable_metric_id when POSTing the request @AlexEven

File: templates/payg/algolia.mdx [hunk 2]
```diff
@@ -2,4 +2,279 @@
...
+            "name": "Algolia committed",
+            "code": "algolia_committed",
+            "interval": "monthly",
+            "amount_cents": 12000,
+            "amount_currency": "USD",
+            "trial_period": 0.0,
+            "pay_in_advance": true,
+            "charges": [
+                {
+                    "lago_billable_metric_id": "15879740-102e-43bd-a6bc-15d865bb1b85",
```
Comment (sarkissianraffi) [inline templates/payg/algolia.mdx:174]: Same comment as above. You should use the billable_metric_id

File: templates/payg/algolia.mdx [hunk 3]
```diff
@@ -2,4 +2,279 @@
...
```
Comment (sarkissianraffi) [inline templates/payg/algolia.mdx:230]: same as above

**Review by sarkissianraffi — CHANGES_REQUESTED** [review]

File: templates/payg/algolia.mdx (hunk 1 above)
Comment (AlexEven) [reply templates/payg/algolia.mdx]: Just updated it, thank you Raffi!

File: templates/payg/algolia.mdx (hunk 2 above)
Comment (AlexEven) [reply templates/payg/algolia.mdx]: Done ✔️

File: templates/payg/algolia.mdx (hunk 3 above)
Comment (AlexEven) [reply templates/payg/algolia.mdx]: Done ✔️

**Review by sarkissianraffi — APPROVED** [review]

**Pull request merged**